tensilelite/Tensile/LibraryLogic.py

```diff
@@ -466,7 +468,13 @@ def addFromCSV(self, dataFileName, numSolutions, solutionMap):
         else:
           printWarning("Performance unit %s in %s is unrecognized: assuming GFlops (device efficiency)" % (perfUnit, dataFileName))
           self.perfMetric = "DeviceEfficiency"
-
+        
+        # get the column index of Frequency(MHz)
+        try:
+          columnOfFreqIdx = row.index(" WinnerFreq")
```

Comment:
to be safer, please consider use the same method to get row index
          columnOfWinnerGFlops = row.index(" WinnerGFlops")
          columnOfWinnerIdx = row.index(" WinnerIdx")

Reply:
Added

Reply:
Added

Reply:
should we rename the " WinnerFreq" as " DeviceMaxFreq" or something else, since it follow the GPU but not winner

Reply:
Sounds reasonable, I have renamed "WinnerFreq" as "DeviceMaxFreq".